examples/applications/ccm/aws-helm/helm-chart.yaml

```
@@ -15,7 +15,7 @@ spec:
         - --use-service-account-credentials=true
         - --configure-cloud-routes=false
       image: |-
-        tag: ${ trimSuffix "+rke2r1" .ClusterValues.Cluster.spec.topology.version }
+        tag: v1.28.3
```

```suggestion
        tag: v1.31.4
```
Can we use this, its more [recent](https://github.com/kubernetes/cloud-provider-aws/releases/tag/v1.31.4), and matches with [KUBERNETES_MANAGEMENT_VERSION](https://github.com/rancher/turtles/blob/main/test/e2e/config/operator.yaml#L44)

---

I would actually go as far as `v1.32.2`, it should work with more recent versions.

---

registry.k8s.io/provider-aws/cloud-controller-manager:v1.32.0 is available. Also nightly is passing. We need to use same version across templates, so existing configuration is enough.

---

I bumped to v1.32.1